File: app/src/main/java/com/askjeffreyliu/myapplication/MyRadioButton.java
package com.askjeffreyliu.myapplication;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;
import android.widget.TextView;

import com.askjeffreyliu.flexboxradiogroup.CustomizableRadioButton;

public class MyRadioButton extends CustomizableRadioButton {

    private TextView textView;

    public MyRadioButton(Context context) {
        super(context);
        initView();
    }

    public MyRadioButton(Context context, AttributeSet attrs) {
        super(context, attrs);
        initView();
    }

    private void initView() {
        View view = inflate(getContext(), R.layout.my_radio_button, null);
        addView(view);
        textView = view.findViewById(R.id.text);
    }

    @Override
    public void setChecked(boolean b) {
        super.setChecked(b);
        if (textView != null) {
            textView.setText(b ? android.R.string.yes : android.R.string.no);
        }
    }
}

File: app/src/main/java/com/askjeffreyliu/myapplication/MyRadioButton.kt
package com.askjeffreyliu.myapplication

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import com.askjeffreyliu.flexboxradiogroup.CustomizableRadioButton
import kotlinx.android.synthetic.main.my_radio_button.view.*

class MyRadioButton @JvmOverloads constructor(
        context: Context,
        attrs: AttributeSet? = null
) : CustomizableRadioButton(context, attrs) {

    init {
        LayoutInflater.from(context)
                .inflate(R.layout.my_radio_button, this, true)
    }

    override fun setChecked(b: Boolean) {
        super.setChecked(b)
        if (text != null) {
            text.setText(if (b) android.R.string.yes else android.R.string.no)
        }
    }
}